src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/MemoryHelpers.cs

```diff
@@ -15,7 +16,8 @@ private static unsafe void MemSet(ref byte dest, byte value, nuint size)
         {
             if (size > 0)
             {
-                _ = dest;
+                // Implicit nullcheck
```

Comment:
Do we have a test that explicitly tests that initblk and cpblk IL instructions throw NullReferenceExceptions?

Reply:
Added